app/assets/stylesheets/static/_index.css.scss

```diff
@@ -5,45 +5,89 @@
   background-position: 50% 30%;
 }
 
-.what-is-tea-time {
-  @include cover-photo(image-url("home/what-is-tws-cover.png"), $gradient: $null-gradient);
-  @include padding(150px 0);
+.what-is-tea-time-wrapper {
+  width: 100%;
+  background: $tws-tan;
+  text-align: center;
+  padding: 2em 1em;
+  @include margin(0 0 3em);
+  .what-is-tea-time {
+    .part {
+      @include margin(0 0 $base-font-size*2);
 
-  @include media($tablet) {
-    @include padding(150px 25px);
+      &.last {
+        @include margin(0);
+      }
+
+      .teatime-basics {
+        @include margin(0 0 $base-font-size/2 0);
+      }
+
+      .teatime-basics-subtext {
+        @include margin(0);
```

Comment:
rather than `@include`-ing a billion classes, if you want no-margin headers create an extend for it